.github/workflows/validate.yml

```diff
@@ -33,3 +33,33 @@ jobs:
           path: |
             tests/test-suite.log
             tests/coverage-*.log
+
+  rpm-tests:
+    runs-on: ubuntu-latest
+    # start from a minimal container and install only our build deps; mock does
```

Comment:
Some further context here: I deliberately do *not* want to run this in the same container (image) as the unit tests -- the latter require a lot more runtime/test dependencies, and I want to test the rpm build in a minimal environment (to avoid missing BuildRequires). I initially tried to do this with mock or a simple chroot inside the container, but that requires too many hacks. We really don't want to run a `--privileged` container, as that would severely hamper test infra portability -- you don't get  privileged containers on shared infrastructure.